### DISCUSSION TITLE
VSCode extension not showing classrooms or assignments has error

### DISCUSSION THREAD

**Discussion by debordem** [community/community#47269] [GitHub Education]
Hi..
I keep getting the error "Refresh: Cannot read properties of null (reading 'id')
I thought it might be something to do with having multiple organisations (for each class I teach or have taught)
As it did show one class, maybe the first one I ever set up.
I went through and deletes all the old classes and now have only one.
I am using the same github account (others suggested this)
I feel the original organisation id is stored somewhere and not able to be changed?
I cant seem to find this anywhere, it might be a VSCode and github issue?
Any help?
M

**Comment by alexpapster** [comment]
@debordem Did you ever figure that out? I am facing the exact same issue

**Reply by paulostradioti** [reply]
Me too.

**Comment by RyzenFromFire** [comment]
I have a similar issue where the "Classrooms" tab simply doesn't show any assignments despite my account having accepted two of them, displaying the message: "You have not accepted any assignments. Accept an assignment sent to you via email or ask your instructor for a link."

**Reply by jeannettekartchner** [reply]
I'm having the same problem

**Comment by davidecavallari** [comment]
Similar issue here. 

It only shows the assignment from github-campus-advisors organization, which was the first assignment I accepted:
<img width="398" alt="image" src="https://github.com/community/community/assets/137596685/2af9d5e7-f002-4d39-91eb-7c40f27d4c55">

It does not show a couple of other assignments (belonging to one organization of mine) that I accepted for testing, though.

**Comment by jsmarier** [comment]
I'd be curious to hear if anyone has a solution. I'm running the same issue. (But this approach worked fine in the past.) For now, I'm asking students to clone the repo as a workaround.
